packages/test/generatedCode/models/microsoft/graph/dateTimeTimeZoneImpl.ts

```diff
@@ -0,0 +1,44 @@
+import {DateTimeTimeZone} from './dateTimeTimeZone';
+import {AdditionalDataHolder, Parsable, ParseNode, SerializationWriter} from '@microsoft/kiota-abstractions';
+
+export class DateTimeTimeZoneImpl implements AdditionalDataHolder, DateTimeTimeZone, Parsable {
```

Comment:
assuming the model interface holds the inheritance to Parsable and AdditionalPropertyHolder, this doesn't need to be repeated here